packages/front-end/components/SafeRollout/Results/ResultsTable.tsx

```diff
@@ -638,3 +704,11 @@ function getChangeTooltip(changeTitle: string, differenceType: DifferenceType) {
   );
   return <>{changeElem}</>;
 }
+
+function Message({ children }: { children: React.ReactNode }) {
+  return (
+    <Flex align="center" justify="center" position="relative" width="100%">
+      {children}
+    </Flex>
+  );
+}
```

Comment:
Small nit: This is only used once in this file and no where else - should we just do this inline?